source/custom/nixnet_service.custom.cpp

```diff
@@ -1,2 +1,207 @@
+#include <nixnet/nixnet_service.h>
+#include <nixnet.pb.h>
+#include <server/converters.h>
+#include "custom/nixnet_converters.h"
+
 namespace nixnet_grpc {
-}  // namespace nixnet_grpc
\ No newline at end of file
+using nidevice_grpc::converters::convert_to_grpc;
+    
+::grpc::Status NiXnetService::ReadState(::grpc::ServerContext* context, const ReadStateRequest* request, ReadStateResponse* response)
```

Comment:
For anything in custom.cpp add a comment to the method explaining why it's there. Also @ni-pkhare

Reply:
Added a comment